conans/client/remote_registry.py

```diff
@@ -169,6 +169,10 @@ def remove(self, remote_name):
             refs = {k: v for k, v in refs.items() if v != remote_name}
             self._save(remotes, refs)
 
+    def clean(self):
+        with fasteners.InterProcessLock(self._filename + ".lock", logger=logger):
```

Comment:
I think it should be 
```suggestion
        with fasteners.InterProcessLock(self._lockfile, logger=logger):
```

Reply:
good catch